tests/pytest/tests.sh

```diff
@@ -90,32 +121,115 @@ valgrind_summary() {
 
 run_tests() {
 	local title="$1"
-	[[ ! -z $title ]] && { $ROOT/opt/readies/bin/sep -0; printf "Tests with $title:\n\n"; }
+	shift
+	if [[ -n $title ]]; then
+		$READIES/bin/sep -0
+		printf "Testing $title:\n\n"
+	fi
+
 	cd $ROOT/tests/pytest
-	[[ ! -z $TESTMOD ]] && RLTEST_ARGS+=--module $TESTMOD
-	$OP python3 -m RLTest --clear-logs --module $MODULE --module-args "JSON_BACKEND SERDE_JSON" $RLTEST_ARGS
-	$OP python3 -m RLTest --clear-logs --module $MODULE $RLTEST_ARGS
+
+	[[ $SERDE_JSON == 1 ]] && MODARGS+=" JSON_BACKEND SERDE_JSON"
+	
+	if [[ $EXISTING_ENV != 1 ]]; then
+		rltest_config=$(mktemp "${TMPDIR:-/tmp}/rltest.XXXXXXX")
+		rm -f $rltest_config
```

Comment:
You shouldn't have to remote this file if you're calling mktemp on it.

Reply:
Right, of course. I really don't know where this came from.